docs/00-project/ai/prompts/library/audit/project/new/README.md

```
@@ -0,0 +1,46 @@
+______________________________________________________________________
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)